conf/config.yaml

```diff
@@ -100,6 +100,12 @@ nginx_config:                     # config for render the template to genarate n
     worker_connections: 10620
   http:
     access_log: "logs/access.log"
+    #access_log_format: |+         # access.log recommended format
+    #  '{"host":"$http_host", "uri":"$uri", "scheme":"$scheme", "rsp_st":"$status", "ups_st":"$upstream_status", "cip":"$remote_addr", '
```

Comment:
we need to add `escape=json`:

http://nginx.org/en/docs/http/ngx_http_log_module.html#log_format